WordPress/src/main/java/org/wordpress/android/ui/stats/service/StatsServiceStarter.java

```diff
@@ -0,0 +1,93 @@
+package org.wordpress.android.ui.stats.service;
+
+import android.annotation.TargetApi;
+import android.app.job.JobInfo;
+import android.app.job.JobScheduler;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.PersistableBundle;
+
+import org.wordpress.android.util.AppLog;
+
+public class StatsServiceStarter {
+    public static final String ARG_START_ID = "start-id";
+    private static final int JOB_STATS_SERVICE_ID = 8000;
```

Comment:
Should we move this ID to an outside class common to the whole app that holds all the IDs of all JobServices in the app?
I see this in the documentation
`jobId - Application-provided id for this job. Subsequent calls to cancel, or jobs created with the same jobId, will update the pre-existing job with the same id. This ID must be unique across all clients of the same uid (not just the same package). You will want to make sure this is a stable id across app updates, so probably not based on a resource ID.`

It seems that the IDs should be unique across the whole app. It's unlikely that we re-use the same ID in other parts of the app at the moment, but isn't better to have a common place with all of them?

(Btw, discard this comment if I got it wrong. I'm not an expert of JobServices yet).

Reply:
You're totally correct @daniloercoli  - that's why I chose to use a different `JOB_ID` for each of the new `JobService`s  - that should be enough for the rest of services as they all just have one request to be served while being used, the only exception being  `StatsService` actually, and that's why the actual `id` given is not just the static field `JOB_STATS_SERVICE_ID` but  actuallly `JOB_STATS_SERVICE_ID + getNewStartId()`, so we should be good to go here

Reply:
addressed taking all the Ids out to a class in 0ff54c5

Reply:
Right, I'm more worried about other services that we can run in other part of the app.